kustomize/base/7.1.0/ingress/ingress.yaml

```diff
@@ -16,60 +15,99 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-buffers: 4 256k
     nginx.ingress.kubernetes.io/proxy-busy-buffers_size: 256k
     nginx.ingress.kubernetes.io/error-log-level: "debug"
+  name: mtls
+spec:
+  rules:
+    - host: $(MTLS_FQDN)
+      http:
+        paths:
+          - backend:
+              service:
+                name: ig
+                port:
+                  number: 80
+            path: /am/oauth2/realms/root/realms/alpha/access_token
+            pathType: Exact
+          - backend:
+              service:
+                name: ig
+                port:
+                  number: 80
+            path: /am/oauth2/realms/root/realms/alpha/register
+            pathType: Exact
+          - backend:
+              service:
+                name: ig
+                port:
+                  number: 80
+            path: /rs/
+            pathType: Prefix
+  tls:
+    - hosts:
+        - $(MTLS_FQDN)
+      secretName: obwac
+---
+# Ingress for access to routes protected with TLS by a ForgeRock cert
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/http2-max-field-size: 16k
+    nginx.ingress.kubernetes.io/http2-max-header-size: 128k
+    nginx.ingress.kubernetes.io/proxy-body-size: 64m
+    nginx.ingress.kubernetes.io/proxy-buffer-size: 128k
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+    nginx.ingress.kubernetes.io/proxy-buffers: 4 256k
+    nginx.ingress.kubernetes.io/proxy-busy-buffers_size: 256k
+    nginx.ingress.kubernetes.io/error-log-level: "debug"
```

Comment:
Should this be controlled by an env var? I suspect this could generate a lot of log entries and only be needed when it's needed if you know what I mean?

Reply:
I can take a look at that, it's tricky in kustomize. You can only use variable replacement in certain places in the template.

Reply:
Raised a separate issue for that: https://github.com/SecureApiGateway/SecureApiGateway/issues/843